### PR TITLE
I18N: Change wording of `roll back`

### PR DESCRIPTION
**Pull request by kebbet**
See ticket description.

https://core.trac.wordpress.org/ticket/58282

### CODE REVIEW

File: src/wp-admin/includes/class-core-upgrader.php [hunk 1]
```diff
@@ -34,8 +34,8 @@ public function upgrade_strings() {
 		$this->strings['unpack_package']        = __( 'Unpacking the update&#8230;' );
 		$this->strings['copy_failed']           = __( 'Could not copy files.' );
 		$this->strings['copy_failed_space']     = __( 'Could not copy files. You may have run out of disk space.' );
-		$this->strings['start_rollback']        = __( 'Attempting to roll back to previous version.' );
-		$this->strings['rollback_was_required'] = __( 'Due to an error during updating, WordPress has rolled back to your previous version.' );
+		$this->strings['start_rollback']        = __( 'Attempting to restore back to previous version.' );
```
Comment (costdev) [inline src/wp-admin/includes/class-core-upgrader.php:37]:
```suggestion
		$this->strings['start_rollback']        = __( 'Attempting to restore the previous version.' );
```

Comment (costdev) [inline src/wp-admin/includes/class-core-upgrader.php]:
```suggestion
		$this->strings['start_rollback']        = __( 'Attempting to restore the previous version.' );
```

File: src/wp-admin/includes/file.php [hunk 2]
```diff
@@ -310,7 +310,7 @@ function wp_print_file_editor_templates() {
 					<?php
 					printf(
 						/* translators: 1: Line number, 2: File path. */
-						__( 'Your PHP code changes were rolled back due to an error on line %1$s of file %2$s. Please fix and try saving again.' ),
+						__( 'Your PHP code changes were restored due to an error on line %1$s of file %2$s. Please fix and try saving again.' ),
```
Comment (costdev) [inline src/wp-admin/includes/file.php]:
```suggestion
						__( 'Your PHP code changes were not applied due to an error on line %1$s of file %2$s. Please fix and try saving again.' ),
```

**Review by costdev — CHANGES_REQUESTED** [review]
Thanks for the PR Kebbet! I've left some suggestions of language I think might work well im each context.

**Comment by kebbet** [comment]
Thanks for the review @costdev, much better with your suggestions! They are now applied.

**Review by audrasjb — APPROVED** [review]

**Comment by SergeyBiryukov** [comment]
The suggested strings indeed look better to me. Thanks!

**Review by SergeyBiryukov — APPROVED** [review]

**Comment by audrasjb** [comment]
committed in https://core.trac.wordpress.org/changeset/55750

**Pull request closed without merging**